README.md

```diff
@@ -34,6 +34,8 @@ needs to be in your current directory when you run `ansible` or
 current ansible configuration; make sure to update the relative paths to the
 foreman-ansible-module `modules` and `module_utils` if you do so.
 
+Also make sure to update `ansible_nailgun_cement.py` in `module_utils` with the nailgun module name when developing new module.
```

Comment:
I think, those lines are not quite up to date. With the move from nailgun to apypie they might introduce more confusion.